x/orderbook/msg.go

```diff
@@ -0,0 +1,104 @@
+package orderbook
+
+import (
+	"github.com/iov-one/weave"
+	coin "github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/migration"
+)
+
+func init() {
+	// Migration needs to be registered for every message introduced in the codec.
+	// This is the convention to message versioning.
+	migration.MustRegister(1, &CreateOrderBookMsg{}, migration.NoModification)
```

Comment:
[weave/migration](../migration) documentation seems like a clear documentation about how migration works. I'll ask you more specific questions in future if needed.